obs-studio-client/source/nodeobs_api.cpp

```diff
@@ -89,11 +89,29 @@ void api::OBS_API_getPerformanceStatistics(const v8::FunctionCallbackInfo<v8::Va
 	    v8::String::NewFromUtf8(args.GetIsolate(), "percentageDroppedFrames").ToLocalChecked(),
 	    v8::Number::New(args.GetIsolate(), response[3].value_union.fp64));
 	statistics->Set(
-	    v8::String::NewFromUtf8(args.GetIsolate(), "bandwidth").ToLocalChecked(),
+	    v8::String::NewFromUtf8(args.GetIsolate(), "streamingBandwidth").ToLocalChecked(),
```

Comment:
This is a breaking change streamlabs-obs client should integrate as part of this release.

Reply:
So get it back to `bandwidth` then?

Reply:
We can totally change it, we just need to correctly apply the change as part of the PR that will update the osn in the frontend.